server/auth/providers/ethereum-provider.ts

```diff
@@ -91,7 +91,13 @@ export const EthereumProvider = (currentToken: JWT, req: NextApiRequest): AuthPr
           networks.map(({ networkAddress, chain_id }) => UserRoleUtils.getGovernorRole(chain_id, networkAddress)))
         .catch(() => []);
 
-      roles.push(...governorOf);
+      const allowBountyCreationOnNetworks = await models.network
+        .findAll({attributes: ["allow_list", "id"],})
+        .then((rows) =>
+          rows.filter(({allow_list}) => (!allow_list.length || allow_list.includes(address)))
+            .map(({id}) => UserRoleUtils.getCreateBountyRole(id)))
+
+      roles.push(...governorOf, ...allowBountyCreationOnNetworks);
```

Comment:
You can filter directly in the query using the operator `[Op.contains]`

Reply:
I thought about that, but then we need to allow bounty creation _if there's no length for the allow_list_ which would be "contains: false" -- hence I decided to just filter them out before mapping